pkg/logger/logger.go

```diff
@@ -0,0 +1,289 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package logger
+
+import (
+	"os"
+	"sync"
+	"time"
+
+	"github.com/dapr/dapr/pkg/version"
+	"github.com/dapr/dapr/utils"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// LogTypeLog is normal log type
+	LogTypeLog = "log"
+	// LogTypeRequest is Request log type
+	LogTypeRequest = "request"
+
+	// Field names that defines Dapr log schema
+	logFieldTimeStamp = "time"
+	logFieldLevel     = "level"
+	logFieldType      = "log_type"
+	logFieldScope     = "scope"
+	logFieldMessage   = "message"
+	logFieldInstance  = "instance"
+	logFieldDaprVer   = "dapr_ver"
+	logFieldDaprID    = "dapr_id"
+)
+
+// LogLevel is Dapr Logger Level type
+type LogLevel int
+
+const (
+	// DebugLevel has verbose message
+	DebugLevel LogLevel = iota
+	// InfoLevel is default log level
+	InfoLevel
```

Comment:
Does DebugLevel enables all Info, Warn, Error, Fatal.
Does INfoLevel enables all Warn, Error, Fatal
and so on

Reply:
Yes correct. it is handled by logrus pkg.